pkg/identity/identity_test.go

```diff
@@ -111,6 +116,15 @@ func (s *IdentityTestSuite) TestAllocateIdentityReserved(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(i.ID, Equals, ReservedIdentityInit)
 	c.Assert(isNew, Equals, false)
+
+	lbls = labels.Labels{
+		labels.IDNameInit: labels.NewLabel(labels.IDNameUnmanaged, "", labels.LabelSourceReserved),
```

Comment:
Why use `IDNameInit` here?

Reply:
Good catch fixed. It was primarily cosmetic but clearly wrong.